vince/models.py

```diff
@@ -3856,6 +3863,74 @@ class CVEAffectedProduct(models.Model):
         blank=True,
         null=True)
 
+class VendorProduct(models.Model):
+    """
+    Store Vendor Product information. 
+    """
+    
+    name = models.CharField(
+    _('Product Name'),
+    max_length=200)
+
+    organization = models.ForeignKey(
+        Contact,
+        on_delete=models.CASCADE,
+        blank=False,
+        null=False)
+    
+    sector = ArrayField( models.CharField( max_length = 50 ), blank = True, null = True )
+    
+    uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
```

Comment:
Shouldn't this have META -  unique_together @mstrad  @z-priest 
```
    class Meta:
        unique_together = (('name', 'organization?'),)
```

Reply:
I was not aware that unique_together was a thing, but that makes sense. In that case, would we need to change this to be UUIDv5? I believe UUIDv5 is deterministically calculated based on the combination of organization and name or does the UUID library somehow do this with UUIDv5?

Reply:
Thank You,

@mstrad I believe this would be correct.

Reply:
Yes that's correct